graph/api/api.go

```diff
@@ -14,7 +16,7 @@ import (
 )
 
 // GraphNamespaces generates a namespaces graph using the provided options
-func GraphNamespaces(business *business.Layer, o graph.Options) (code int, config interface{}) {
+func GraphNamespaces(business *business.Layer, o graph.Options, ctx context.Context) (code int, config interface{}) {
```

Comment:
Conventionally context is passed as the first argument

Reply:
Fixed.